Formula/gmic.rb

```diff
@@ -1,8 +1,8 @@
 class Gmic < Formula
   desc "Full-Featured Open-Source Framework for Image Processing"
   homepage "https://gmic.eu/"
-  url "https://gmic.eu/files/source/gmic_3.0.1.tar.gz"
-  sha256 "6cc20a20e3ab53ce485ccf6e044a30141b3d62cf7743b83bb04906ff29453035"
+  url "https://gmic.eu/files/source/gmic_3.1.5.tar.gz"
+  sha256 "fa77e85b3a39638008515ac525f23f0ed7a45b63c463c0ba6292c87f5e88e30d"
```

Comment:
Alternative option would be to use CMake scripts from Github generated tarball or as resource.
```suggestion
  url "https://github.com/dtschump/gmic/archive/refs/tags/v.3.1.5.tar.gz"
  sha256 "8e62e2fdddc26658d4fedd41bc0664217ad07823fca86cd59e2334f14d1ecb02"
```

And then may need to move files during install:
```rb
buildpath.install "resources/CMakeLists.txt"
buildpath.install "resources/cmake"
```